deploy/webhooks.yaml

```diff
@@ -152,4 +152,52 @@ webhooks:
     resources:
     - ssps
   sideEffects: None
-  timeoutSeconds: 10
\ No newline at end of file
+  timeoutSeconds: 10
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: hostpath-provisioner-operator-webhook-service-cert
+  labels:
+    name: hostpath-provisioner-operator
```

Comment:
```suggestion
    name: kubevirt-hyperconverged
```

Reply:
Same on line 177?

Reply:
No, I think only here

Reply:
Okay, note the ssp one above doesn't have it
```yaml
 apiVersion: cert-manager.io/v1
 kind: Certificate
 metadata:
   name: ssp-operator-service-cert
   labels:
     name: ssp-operator <-------
 spec:
   secretName: ssp-operator-service-cert
   dnsNames:
   - ssp-operator-service.kubevirt-hyperconverged.svc
   issuerRef:
     name: selfsigned

```